lib/CL/devices/devices.c

```diff
@@ -114,6 +142,43 @@ static unsigned devices_active = 0;
 
 static pocl_lock_t pocl_init_lock = POCL_LOCK_INITIALIZER;
 
+static void * pocl_device_handles[POCL_NUM_DEVICE_TYPES];
+
+static void
+getPoclDeviceLibPath (char *result, char *device_name)
```

Comment:
Check the style of this name. This is a C file, thus the C style applies; pls use underscores instead.